syntax = "proto3";

message ReqCreateRoom {
  uint64 request_id = 1;
}

message ResCreateRoom {
  uint64 request_id = 1;
  bool success = 2;
  uint64 room_id = 3;
}

message ReqJoinRoom {
  uint64 request_id = 1;
  uint64 room_id = 3;
}

message ResJoinRoom {
  uint64 request_id = 1;
  bool success = 2;
  uint64 room_id = 3;
}

message ReqFwdRoom {
  uint64 request_id = 1;
  string message = 2;
  repeated string users = 3;
}

message ResFwdRoom {
  uint64 request_id = 1;
  string message = 2;
  repeated string users = 3;
  bool success = 4;
}
